.github/workflows/lint.yml

```diff
@@ -28,3 +28,14 @@ jobs:
           github-token: ${{ secrets.github_token }}
         # Check only if there are differences in the source code
         if: "env.GIT_DIFF"
+  solhint:
```

Comment:
can you also bump the line 26 version too from `version: v1.42.1` to  `version: 1.44`